defrag/modules/helpers/caching.py

```diff
@@ -42,8 +42,12 @@ class Store:
     async def fetch_items() -> Optional[List[Any]]:
```

Comment:
I think you should rebase this on latest dev, because this file has been renamed in some past commit

Reply:
should be ok now